SimpleInventoryManagementSystem/Inventory.cs

```diff
@@ -1,15 +1,37 @@
 ﻿
 using System;
+using SimpleInventoryManagementSystem;
 
-namespace ISimpleInventoryManagementSystem
+
+namespace SimpleInventoryManagementSystem;
+public class Inventory
 {
-    public class Inventory
+    private List<Product> _products;
+    private const string _currency="$";
+    private InputLogger _input;
+
+    public Inventory(ICollection<Product> products)
+    {
+        _products = (List<Product>) products;
+        _input = new InputLogger();
+    }
+
+    public  void AddProduct()
     {
-        private List<Product> products;
+        Product product=_input.ProductInput();
```

Comment:
Try to use the format on save function on your IDE so your code gets auto formatted

```suggestion
        Product product = _input.ProductInput();
```